webv2/updateTableSchema/reviewrenameColumn.go

```diff
@@ -0,0 +1,216 @@
+package updateTableSchema
+
+import (
+	"github.com/cloudspannerecosystem/harbourbridge/internal"
+	"github.com/cloudspannerecosystem/harbourbridge/spanner/ddl"
+)
+
+// reviewRenameColumn review  rename Columnname in schmema.
+func reviewRenameColumn(newName, table, colName string, Conv *internal.Conv, interleaveTableSchema []InterleaveTableSchema) []InterleaveTableSchema {
+
+	sp := Conv.SpSchema[table]
+
+	columnId := sp.ColDefs[colName].Id
+
+	// update foreignKey relationship Table column names
+	for i, _ := range sp.Fks {
+
+		reviewRenameColumnNameInForeignkeyTableSchema(Conv, sp, i, colName, newName)
+
+	}
+
+	for _, sp := range Conv.SpSchema {
+
+		for j := 0; j < len(sp.Fks); j++ {
+			if sp.Fks[j].ReferTable == table {
+
+				reviewRenameColumnNameInForeignkeyReferTableSchema(Conv, sp, sp.Name, colName, newName)
+			}
+
+		}
+
+	}
+
+	// update interleave table relation
+	isParent, parentSchemaTable := IsParent(table)
+
+	if isParent {
+
+		interleaveTableSchema = reviewRenameColumnNameInparentTableSchema(Conv, parentSchemaTable, interleaveTableSchema, colName, newName)
+	}
+
+	childSchemaTable := Conv.SpSchema[table].Parent
+
+	if childSchemaTable != "" {
+
+		interleaveTableSchema = reviewRenameColumnNameInchildTableSchema(Conv, childSchemaTable, interleaveTableSchema, colName, newName)
+
+	}
+
+	interleaveTableSchema = reviewreanmeColumnNameInCurrentTable(Conv, sp, interleaveTableSchema, table, columnId, colName, newName, parentSchemaTable, childSchemaTable)
+
+	return interleaveTableSchema
+
+}
+
+// reviewRenameForeignkeyTableSchema review  rename Columnname in Foreignkey Table Schema.
+func reviewRenameColumnNameInForeignkeyTableSchema(Conv *internal.Conv, sp ddl.CreateTable, index int, colName string, newName string) {
```

Comment:
same issue here as mentioned in renameColumn.go

Reply:
Updated to use a common function